setup.cfg

```diff
@@ -0,0 +1,46 @@
+[metadata]
+license_file = LICENSE
+long_description = file: README.md
+
+[options]
+install_requires =
+    grpcio==1.20.1
+    grpcio-tools==1.20.1
+    protobuf==3.7.0
+    aio-pika==3.0.1
+    aiohttp==3.4.4
+    coloredlogs==10.0
+    earlgrey==0.0.4
+    fluent-logger==0.9.3
+    gunicorn==19.9.0
+    jsonrpcclient==2.6.0
+    jsonrpcserver==3.5.6
+    leveldb==0.20
+    plyvel==1.1.0
+    pika==0.12.0
+    python-pkcs11==0.5.0
+    requests==2.20.0
+    cryptography==2.3.1
+    sanic==18.12.0
+    secp256k1==0.13.2
+    setproctitle==1.1.10
+    transitions==0.6.8
+    verboselogs==1.7
+    websockets==6.0
+    yappi==0.98
+    dataclasses; python_version < '3.7'
+
+[options.extras_require]
+icon =
+    iconservice
+    iconcommons
+    iconrpcserver
+    tbears
+icon_dev =
+    iconservice @ git+https://github.com/icon-project/icon-service.git@develop
+	iconcommons @ git+https://github.com/icon-project/icon-commons.git@master
+	iconrpcserver @ git+https://github.com/icon-project/icon-rpc-server.git@develop
```

Comment:
Please use unified indent type(`tab` or `space`?)
I recommend `space` if it has no problem.

Reply:
Thank you.